source/configuration_manual/shared_mailboxes/cluster_setup.rst

```diff
@@ -122,3 +122,18 @@ possibilities for it are:
 
 -  Any other `shared dictionary <https://wiki.dovecot.org/Dictionary>`__ can
    be used like described at :ref:`user_shared_mailboxes_shared_mailbox_listing`
+
+
+Shared folders and quota
+------------------------
+
+When using the :ref:`quota_plugin` it must be configured to not attempt to
+calculate quota for the shared namespaces. There are multiple ways this can be
+achieved.
+
+#. Configure the first ``quota_rule`` to ignore the shared namespace configured:
+     ``quota_rule = <shared namespace prefix>*:ignore``.
+#. Configure quota to exclusively calculate quota for the private namespace in
+     use ``quota=count:ns=<inbox namespace prefix>``
```

Comment:
indentation doesn't look right in here. does it work?..

Reply:
It looked okay but wasn't correct.